src/Views/AppInfoView.vala

```diff
@@ -351,7 +342,6 @@ namespace AppCenter.Views {
             header_grid.attach (package_name, 1, 0);
             header_grid.attach (package_author, 1, 1, 2);
             header_grid.attach (origin_combo_revealer, 1, 2, 2);
-            header_grid.attach (app_version, 2, 0);
```

Comment:
While you're here, the other grid attachments should be updated; the author and origin combo don't need to be two columns wide anymore.